include/zcbor_decode.h

```diff
@@ -107,6 +107,24 @@ bool zcbor_float32_64_expect(zcbor_state_t *state, double expected); /* IEEE754
 bool zcbor_float64_expect(zcbor_state_t *state, double expected); /* IEEE754 float64 */
 bool zcbor_float_expect(zcbor_state_t *state, double expected); /* IEEE754 float16, float32, or float64 */
 
+/** Like the _expect() functions but with a pointer value if it was not already a pointer.
```

Comment:
nit: I don't follow this description. Does the following boil down to the same, or am I missing something?
```
/** Like the _expect() functions but the value is passed through a pointer.
```

Reply:
Good improvement, thanks!